Sources/UtiliKit/General/LanguageTool.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LanguageTool.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 10/11/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
+@available(iOSApplicationExtension 11.0, *)
+class LanguageTool {
+
+    private let badverbs = ["be", "have", "do", "say", "get", "make", "go", "know", "take", "see", "come", "think", "look", "want", "give", "use", "find", "tell", "ask", "work", "seem", "feel", "try", "leave", "call"]
```

Comment:
We may want to camelcase these to be consistent with the `badOthers` property name below.

```suggestion
    private let badVerbs = ["be", "have", "do", "say", "get", "make", "go", "know", "take", "see", "come", "think", "look", "want", "give", "use", "find", "tell", "ask", "work", "seem", "feel", "try", "leave", "call"]
```